zipkin-lens/src/components/DependenciesPage/DependenciesPageHeader.jsx

```diff
@@ -105,7 +108,10 @@ const DependenciesPageHeader = React.memo(
             value={startTime}
             onChange={onStartTimeChange}
             className={classes.dateTimePicker}
-            InputProps={{ classes: { input: classes.dateTimePickerInput } }}
+            InputProps={{
+              disabled: true,
```

Comment:
same file, so make a constant and use in both places? then you have only one place to add comment why